resources/assets/less/bem/bootstrap/modal.less

```diff
@@ -0,0 +1,50 @@
+/**
+*    Copyright 2015 ppy Pty. Ltd.
+*
+*    This file is part of osu!web. osu!web is distributed with the hope of
+*    attracting more community contributions to the core ecosystem of osu!.
+*
+*    osu!web is free software: you can redistribute it and/or modify
+*    it under the terms of the Affero GNU General Public License as published by
+*    the Free Software Foundation, either version 3 of the License, or
+*    (at your option) any later version.
+*
+*    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+*    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+*    See the GNU Affero General Public License for more details.
+*
+*    You should have received a copy of the GNU Affero General Public License
+*    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+*
+*/
+
+.modal {
+	// FIXME: Chrome sometimes fail painting the modal box without this
+	backface-visibility: hidden;
+
+	// Elements
+	&__dialog {
+		width: 430px;
+		color: @text-color;
+	}
+
+	// Modifiers
+}
+
+// Javascript-based hooks/styling
+.js-user-dropdown-modal { 
```

Comment:
Hm, you might be right, however I can't seem to think of a more suitable place.

It's related to the modal as it [already was] (https://github.com/ppy/osu-web/blob/master/resources/assets/less/login-modal.less), so I think this should be okay-ish?